smarts/core/scenario.py

```diff
@@ -397,8 +397,9 @@ def setdefault(l: Sequence[Any], index: int, default):
             social_agents_info.append(
                 {agent.id: (agent, mission) for agent, mission in l}
             )
-
+        # pytype: disable=bad-return-type
         return social_agents_info
+        # pytype: enable=bad-return-type
```

Comment:
I believe this could be fixed by changing the return signature to `Sequence[Dict[str, Tuple[SocialAgent, Mission]]].